sdk/ai/azure-ai-inference/setup.py

```diff
@@ -38,7 +38,7 @@
     url="https://github.com/Azure/azure-sdk-for-python/tree/main/sdk",
     keywords="azure, azure sdk",
     classifiers=[
-        "Development Status :: 4 - Beta",
+        "Development Status :: 7 - Inactive",
```

Comment:
Is the GitHub UI showing me the wrong thing, or is there still a setup.py file under ai/azure-ai-inference?

Reply:
This is correct. I'm disabling builds of azure-ai-inference client library (which is already in main branch), since if this PR gets approved, it means we need to ship azure-ai-runtime-inference instead. I don't want to build both here, to speed things up. If we ship azure-ai-runtime-inference, I will delete all files related to azure-ai-inference.